tldw_Server_API/tests/Ingestion_Sources/test_media_sink.py

```
@@ -31,3 +31,44 @@ def test_media_sink_updates_existing_binding_as_new_version(fake_media_db):
+
+    calls: list[dict[str, object]] = []
+
+    class _FakeMediaRepository:
+        def add_media_with_keywords(self, **kwargs):
+            calls.append(kwargs)
+            return 91, "repo-uuid", "created"
+
+    monkeypatch.setattr(
+        media_sink,
+        "get_media_repository",
+        lambda media_db: _FakeMediaRepository(),
```

Ruff (0.15.6)

[warning] Unused lambda argument: `media_db`

(ARG005)